app/jobs/base.rb

```diff
@@ -161,6 +161,15 @@ def self.ensure_interval_logging!
 
     include Sidekiq::Worker
 
+    def self.cluster_concurrency(val)
+      raise ArgumentError, "cluster_concurrency must be 1 or nil" if val != 1 && val != nil
```

Comment:
Maybe we could use `#disable_cluster_concurrency` and `#enable_cluster_concurrency` to avoid the need for a runtime error.